### PR TITLE
docs: fix info style

### PR DESCRIPTION
**Pull request by virkt25**
Fixes how this page is rendered on the docs: https://loopback.io/doc/en/lb4/Parsing-requests.html

Tested in Chrome. Verified all pages I changed to make sure none others broke (after making the usage consistent).

Before: 
image

After:
image

## Checklist

- [x] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

### CODE REVIEW

File: docs/site/Parsing-requests.md [hunk 1]
```diff
@@ -43,10 +43,10 @@ and raw data is parsed from request according to the specification. In the
 example above, the first parameter is from source `path`, so its value will be
 parsed from a request's path.
 
-{% include note.html title="Controller documentation" content="See [controller](Controller.md) for more details of defining an endpoint.
+{% include note.html title="Controller documentation" content=" See [controller](Controller.md) for more details of defining an endpoint.
```
Comment (bajtos) [inline docs/site/Parsing-requests.md]: I find it weird to see a leading space after an opening `"`. How about moving the content to a new line instead?

```
{% include note.html title="Controller documentation" content="
See [controller](Controller.md) for more details of defining an endpoint.
" %}
```

I think this is the style we are already using in other places too.

The same comment applies to other changes made in this pull request.

Comment (virkt25) [reply docs/site/Parsing-requests.md]: I searched all lb4 docs and we're using both patterns, a leading space and new lines.

Mode Controller for example:

```
{% include note.html title="Model documentation" content="See [model](Model.md) to know more details about how to decorate a model class.
" %}
```

 I'm more curious as to why that's needed but don't want to enter Jekyll/Liquid land. I can switch it all to be new lines.

**Comment by virkt25** [comment]
This change fixes the rendering of the `info` / `note.html` on the site. I verified by building a local version of the site: `npm run build:site` and then `cd sandbox/loopback.io && npm start` and viewing the generated pages to confirm the fix.

I'll attach a screenshot after the change to newline.

File: docs/site/Parsing-requests.md (hunk 1 above)
Comment (bajtos) [reply docs/site/Parsing-requests.md]: > I searched all lb4 docs and we're using both patterns, a leading space and new lines.

Fair enough 👍 

In which case, feel free to use whatever style you like more.

**Pull request merged**